src/System.Private.ServiceModel/pkg/System.Private.ServiceModel.pkgproj

```diff
@@ -2,15 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
-  <PropertyGroup>
-    <PreventImplementationReference>true</PreventImplementationReference>
-  </PropertyGroup>
-
   <ItemGroup>
-    <ProjectReference Include="..\src\System.Private.ServiceModel.builds">
-      <AdditionalProperties>FilterToOSGroup=Windows_NT</AdditionalProperties>
-      <UndefineProperties>%(ProjectReference.UndefinedProperties);PackageTargetFramework</UndefineProperties>
-    </ProjectReference>
+    <ProjectReference Include="..\src\System.Private.ServiceModel.csproj" />
```

Comment:
You shouldn't include this here.  This is the reference package.  Since this is a private assembly we don't want it to be reference-able by anyone.  See [System.Private.Uri.pkgproj](https://github.com/dotnet/corefx/blob/master/src/System.Private.Uri/pkg/System.Private.Uri.pkgproj) for an example.